vn.hoabinh.quan.CoBang.yaml

```diff
@@ -0,0 +1,71 @@
+app-id: vn.hoabinh.quan.CoBang
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+command: cobang
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=host
+  - --device=all
+
+modules:
+
+  - flatpak/libzbar.yml
+
+  - name: python3-wheel
+    buildsystem: simple
+    build-commands: [
+      'pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} wheel'
+    ]
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/75/28/521c6dc7fef23a68368efefdcd682f5b3d1d58c2b90b06dc1d0b805b51ae/wheel-0.34.2.tar.gz
+        sha256: 8788e9155fe14f54164c1b9eb0a319d98ef02c160725587ad60f14ddc57b6f96
+
+  - flatpak/generated-poetry-sources.yml
+
+  - name: gst-plugins-base
```

Comment:
Isn't gst (and modules) already included in the runtime?

Reply:
I cannot find list of softwares included in the runtime. Please give me a link.

Reply:
The Freedesktop.org Runtime contains the stuff here: https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/wikis/Release-Contents

They're working on improving the way this is generated and making it more obvious to find. More specifically for this case:

https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/jobs/574127515/artifacts/file/platform-manifest/usr/manifest.md 

Has gstreamer and a bunch of other stuff in it.

Reply:
If I add these to _meson.build_:

```meson
dependency('gstreamer-plugins-base-1.0')
dependency('gstreamer-plugins-good-1.0')
```

I got this error when building with `flatpak-builder`:

```
Run-time dependency gobject-introspection-1.0 found: YES 1.64.1
Run-time dependency gtk+-3.0 found: YES 3.24.20
Run-time dependency gstreamer-plugins-base-1.0 found: YES 1.16.2
Run-time dependency gstreamer-plugins-good-1.0 found: NO (tried pkgconfig and cmake)

meson.build:30:0: ERROR: Dependency "gstreamer-plugins-good-1.0" not found, tried pkgconfig and cmake
```

Reply:
Found that the app still run without those two dependency lines (but still need to build gst-python)..